src/Acuminator/Acuminator.Vsix/Commands/Formatter/BqlFormatter.cs

```diff
@@ -54,5 +52,20 @@ public static BqlFormatter FromTextView(IWpfTextView textView)
 
 			return new BqlFormatter(newLineCharacter, !convertTabsToSpaces, tabSize, indentSize);
 		}
+
+		private static IEnumerable<SyntaxTrivia> GetUseTabsModeIndentTrivias(int indentSize, int tabSize)
```

Comment:
What is this fix and how it is related to the migration?

Reply:
This fix was needed due to removal of Prepend/Append collection helpers in Acuminator.Utilities. 
Somehow this methods are included in Linq version used by .Net Standart but not included into .Net 4.7.2 which is used in VSIX.  Therefore there was a compilation error in Net standard due to ambigous method call - so I removed our helpers but there was a call to Prepend here.
I didn't want to move just one utility method to VSIX because it was used only in this place so I created this method